modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1116 @@
+
+=== A11ychecker 3.0.2
+
+The {productname} 6.1 release includes an accompanying release of the **A11ychecker** premium plugin.
+
+**A11ychecker** 3.0.2 fixes the following UI bug:
+
+The *Tools > Accessibility Checker* menu item opens a dialog. As such, its menu title should be *Accessibility Checker…*.
+
+The trailing ellipsis indicates the action will not execute without further user input and is required for all menu items that present dialogs. And, previously, this menu item did not.
+
+This update corrects the menu title. *Tools > Accessibility Checker…* now includes the required ellipsis and presents as expected.
+
+For information on the **A11ychecker** plugin, see: xref:a11ychecker.adoc[A11ychecker plugin].
+
+
+=== Comments 3.1.0
+
+The {productname} 6.1 release includes an accompanying release of the **Comments** premium plugin.
+
+NOTE: Enabling any of these features depends on changes made to the Core {productname} editor. As a consequence, version 3.1.0 of the xref:comments.adoc[Comments Premium Plugin] requires {productname} 6.1 or later. 
+
+**Comments** 3.1.0 introduces the following enhancements:
+
+
+==== A new event log API
+
+A new API, `editor.plugins.tinycomments.getEventLog()` is now available.
+
+As its name suggests, it returns a log of all Comment-related events, including:
+
+* when a new comment is added (and who added it).
+
+* when a comment is edited (and who edited it).
+
+* when a reply to a comment is added (and who added the reply).
+
+* when a comment is resolved (and who resolved the comment).
+
+* when a comment is deleted (and who deleted the comment).
+
+The new API also supports an `after` option.
+
+[source,js]
+----
+editor.plugins.tinycomments.getEventLog(
+  { after: '<iso-8601-format-time-stamp>' }
+)
+----
+
+restricts the returned list to Comment events after the time-stamp date.
+
+Underlying this new API is a new `CommentChange` event, which gets fired when any modification is made to Comments data.
+
+This new event also exposes the `getEventLog()` API. For example:
+
+[source,js]
+----
+editor.on('CommentChange', (e) => {
+  const logs = e.getEventLogs();
+});
+----
+
+
+==== Avatars can now be displayed next to usernames in comments
+
+{productname} now generates a different default avatar icon for each user.
+
+And this default icon, presented next to user names in comments, can be replaced with a custom avatar.
+
+In embedded mode, the avatar is supplied by the `tinycomments_author_avatar` option:
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea', // change this value according to your HTML
+  plugins: 'tinycomments',
+  tinycomments_mode: "embedded",
+  tinycomments_author: "testuser",
+  tinycomments_author_name: "Test User",
+  tinycomments_author_avatar: "https://example.com/image.ext"
+});
+----
+
+In callback mode, the avatar is supplied via the `tinycomments_create` and `tinycoments_reply` callbacks.
```

Comment:
```suggestion
In callback mode, the avatar is supplied via `authorAvatar` property in the `tinycomments_create` and `tinycoments_reply` callbacks.
```
Maybe something like this so that it's clear what property name to use?